src/main/scala/com/gu/invoicing/refundErroneousPayment/Program.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.invoicing.refundErroneousPayment
+
+import com.gu.invoicing.refundErroneousPayment.Impl._
+import com.gu.invoicing.refundErroneousPayment.Model._
+
+object Program extends App {
```

Comment:
extends App but does it do anything on its own apart from define a function?

Reply:
That was left over from my local changes.  Removed now.